From 37b54d74a10ef9e139905fa24097a7c7e0a1768f Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Sun, 11 Mar 2012 13:58:03 +0530
Subject: [PATCH 192/609] ARM: OMAP2+: ehrpwm: Removes registration of
 EHRPWM.2

Original commit: 4d9c90b385521a84a085b38aeabdbf29934cee3a

  From git://arago-project.org/git/projects/linux-am33x.git
  And ti-sdk-am335x-evm-05.05.00.00-Linux-x86-Install image

Removes registration of EHRPWM instance 2 to support registration using
new API.

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index ce2f462..9fab3a8 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -976,7 +976,6 @@ static void uart2_init(int evm_id, int profile)
 static void haptics_init(int evm_id, int profile)
 {
 	setup_pin_mux(haptics_pin_mux);
-	register_ehrpwm(HAPTICS_MAX_FREQ);
 	return;
 }
 
-- 
1.7.5.4

